client/resource_attribute_restriction.go

```diff
@@ -44,6 +44,14 @@ func ResourceAttributeRestriction() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"mandatory": {
+				Type:     schema.TypeBool,
+				Optional: true,
```

Comment:
is this right?

Reply:
Yes, you don't need to set it and it will default to false.